ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -1237,6 +1301,52 @@ extension RCTConvert {
             "video": toJson(bufferLevel: bufferLevels.video, mediaType: "video"),
         ]
     }
+
+    static func toJson(httpRequestType: HttpRequestType) -> String? {
+        switch httpRequestType {
+        case .drmLicenseFairplay:
+            return "drm/license/fairplay"
+        case .drmCertificateFairplay:
+            return "drm/certificate/fairplay"
+        case .mediaThumbnails:
+            return "media/thumbnails"
+        case .mediaSubtitles:
+            return "media/subtitles"
+        case .manifestHlsMaster:
+            return "manifest/hls/master"
+        case .manifestHlsVariant:
+            return "manifest/hls/variant"
+        case .keyHlsAes:
+            return "key/hls/aes"
+        case .unknown:
+            return "unknown"
+        default:
+            return nil
+        }
+    }
```

Comment:
This can be simplified!
```suggestion
    static func toJson(httpRequestType: HttpRequestType) -> String {
        return httpRequestType.rawValue
    }
```

Reply:
Amazing 🤩 , thanks for catching this! I really had an oversight here 😅